Server/go.sum

```diff
@@ -0,0 +1,45 @@
+github.com/BurntSushi/freetype-go v0.0.0-20160129220410-b763ddbfe298/go.mod h1:D+QujdIlUNfa0igpNMk6UIvlb6C252URs4yupRUV4lQ=
```

Comment:
[ Need reviews] Not sure if this file needs to be commited as well.

Reply:
https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions

Ensure your go.sum file is committed along with your go.mod file.